.devcontainer/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM nvcr.io/nvidia/isaac-sim:2022.2.0
```

Comment:
This doesn't have the full omniverse suite but it will still launch isaac sim. 
We should figure out how to incorporate the other tools so we can use cache and nucleus